### PR TITLE
bug fix: StatOptions component keyboard event

### PR DESCRIPTION
**Pull request by clhenrick**
@mukhtyar this seems to fix the bug with the Stats keyboard event listener, let me know if it works for you.

### CODE REVIEW

**Comment by mukhtyar** [comment]
@clhenrick perfect! The `await tick()` does the trick. Thanks.

**Comment by clhenrick** [comment]
@mukhtyar so should we merge this one? If so I will add a code comment before merging.

**Review by mukhtyar — APPROVED** [review]
Looks good. We can merge this in.

**Pull request merged**
